lib/dolly/document.rb

```diff
@@ -51,7 +51,8 @@ def rev= value
       doc['_rev'] = value
     end
 
-    def save
+    def save options = {}
+      return false unless options[:validate] == false || valid?
```

Comment:
> what @javierg said.
I don't get the point. Following a convention doesn't mean we're not working "independently" from rails IMO.

TL;DR What @netlogics  said.

This PR only standardizes the return value of the `save` method to a boolean. In case you have some sort of validation in place (which responds to `valid?`) you'll be fine. The option to bypass validations was exactly to give a wider range of devs the option to save without validating in a per basis case.